lib/safira_web/controllers/cv_controller.ex

```diff
@@ -21,29 +21,23 @@ defmodule SafiraWeb.CVController do
     company = Accounts.get_company!(company_id)
 
     if Accounts.is_admin(conn) or curr_company_id == company.id do
-      attendees_cvs =
+      zip =
         Accounts.list_company_attendees(company_id)
         |> Enum.filter(fn x -> x.cv != nil end)
-        |> Enum.map(fn x -> {x.nickname, CV.storage_dir(x)} end)
-        |> Enum.map(fn {nickname, storage_dir} ->
-          {nickname,
-           File.ls!(storage_dir)
-           # should never return nil because we enure the attendee has a CV
-           |> List.first()
-           |> (fn file_name -> Path.join(storage_dir, file_name) end).()
-           |> File.read()}
+        |> Enum.map(fn x ->
+          Zstream.entry(
+            x.nickname <> ".pdf",
+            CV.url({x.cv, x})
+            |> (fn url -> SafiraWeb.Endpoint.url() <> url end).()
```

Comment:
```suggestion
            |> (fn url -> System.get_env("CV_URL", "") <> url end).()
```
Add this new environment variable in `.env.sample`. For testing it must be `http://localhost:4001` and for dev `http://localhost:4000`. For staging and production must be empty.

This is because the `CV.url` function returns the full URL when the file is uploaded to S3, but only the path if it is local